src/Service/ServiceListenerFactory.php

```diff
@@ -81,24 +85,35 @@ class ServiceListenerFactory implements FactoryInterface
             'ViewTemplateMapResolver'        => 'Zend\Mvc\Service\ViewTemplateMapResolverFactory',
             'ViewTemplatePathStack'          => 'Zend\Mvc\Service\ViewTemplatePathStackFactory',
             'ViewPrefixPathStackResolver'    => 'Zend\Mvc\Service\ViewPrefixPathStackResolverFactory',
+            'Zend\View\Renderer\PhpRenderer' => ViewPhpRendererFactory::class,
+            'Zend\View\Strategy\PhpRendererStrategy' => ViewPhpRendererStrategyFactory::class,
+            'Zend\View\View'                 => ViewFactory::class,
         ],
         'aliases' => [
-            'Configuration'                              => 'Config',
+            'Config'                                     => 'config',
+            'Configuration'                              => 'config',
+            'configuration'                              => 'config',
             'Console'                                    => 'ConsoleAdapter',
-            'Di'                                         => 'DependencyInjector',
-            'Zend\Di\LocatorInterface'                   => 'DependencyInjector',
+            'ConsoleDefaultRenderingStrategy'            => View\Console\DefaultRenderingStrategy::class,
+            'HttpDefaultRenderingStrategy'               => View\Http\DefaultRenderingStrategy::class,
+            'View'                                       => 'Zend\View\View',
+            'ViewPhpRendererStrategy'                    => 'Zend\View\Strategy\PhpRendererStrategy',
+            'ViewPhpRenderer'                            => 'Zend\View\Renderer\PhpRenderer',
+            'ViewRenderer'                               => 'Zend\View\Renderer\PhpRenderer',
             'Zend\Form\Annotation\FormAnnotationBuilder' => 'FormAnnotationBuilder',
             'Zend\Mvc\Controller\PluginManager'          => 'ControllerPluginManager',
             'Zend\Mvc\View\Http\InjectTemplateListener'  => 'InjectTemplateListener',
+            'Zend\View\Renderer\RendererInterface'       => 'Zend\View\Renderer\PhpRenderer',
             'Zend\View\Resolver\TemplateMapResolver'     => 'ViewTemplateMapResolver',
             'Zend\View\Resolver\TemplatePathStack'       => 'ViewTemplatePathStack',
             'Zend\View\Resolver\AggregateResolver'       => 'ViewResolver',
             'Zend\View\Resolver\ResolverInterface'       => 'ViewResolver',
-            'ControllerManager'                          => 'ControllerLoader',
         ],
+        /*
```

Comment:
I'm still trying to decide what to do with it; forms will likely be optional, but we don't have a documented way to register the abstract factory yet. Once I have a plan, yes, we'll likely remove it.